src/styles/blocks/stars.css

```
@@ -0,0 +1,34 @@
+.card__stars {
+  display: flex;
+  align-items: flex-end;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../../images/star-active.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--4 .stars__star:last-child {
+  background-image: url(../../images/star.svg);
+  background-repeat: no-repeat;
+}
```

you have to use your ready code from star task
```suggestion
.stars {
  display: flex;
}

.stars__star {
  width: 16px;
  height: 16px;
  margin-right: 4px;
  background-image: url(images/star.svg);
  background-repeat: no-repeat;
  background-position: center;
}

.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(n) {
  background-image: url(images/star-active.svg);
}
```